src/components/ProgressBar.js

```diff
@@ -69,12 +73,24 @@ ProgressBar.propTypes = {
   /**
    * Optionnal className
    */
-  className: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
+  className: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+
+  /**
+   * Whether or not to control the progress from props
+   */
+  controlledProgress: PropTypes.bool,
+
+  /**
+   * Controlled progress value
+   */
+  progress: PropTypes.number,
 };
 
 ProgressBar.defaultProps = {
   type: TYPE.DEFAULT,
-  hide: false
+  hide: false,
+  controlledProgress: false,
+  progress: 0,
```

Comment:
If we rely on progress we need to unset it, so it will be undefined or we can use null.
```suggestion
  progress: null // or not set at all,
```